species/fit/retrieval.py

```diff
@@ -787,6 +796,9 @@ def run_multinest(
         fit_corr: Optional[List[str]] = None,
         cross_corr: Optional[List[str]] = None,
         n_live_points: int = 2000,
+        const_efficiency_mode: Optional[bool] = True,
```

Comment:
Thanks, these parameters are useful to include indeed! Perhaps we could add a `multinest_kwargs` parameter? So dictionary which then can also includes `n_live_points` and `resume`?

Reply:
Perhaps for now leave in the `resume` and `n_live_points` though. We can add a deprecation warning.

Reply:
I added `mutlinest_kwargs` and a deprecation warning for `resume` and `n_live_points`, but am not sure if it is implemented how you intended. Let me know!

Reply:
This looks great, thanks!